corehq/util/tests/test_couch.py

```diff
@@ -76,3 +77,121 @@ def test_get_document_or_404_success(self):
         """
         doc = get_document_or_404(MockModel, 'ham', '123')
         self.assertEqual(doc, {'wrapped': {'_id': '123', 'domain': 'ham', 'doc_type': 'MockModel'}})
+
+
+class LoggingDB(object):
```

Comment:
Ah, no I didn't.  Although it looks like it doesn't support bulk operations yet, and I mostly needed this to log the requests made to the db, so I probably would've had to make additions there anyways.